app/lib/cds_importer/entity_mapper/base_mapper.rb

```diff
@@ -164,7 +164,7 @@ def parse
               # iterating through all items in Array and creating @values copy
               value.each do |v|
                 # [1,2,3] => {1=>{2=>{3=>value}}
-                tmp = current_path.lazy.reverse_each.inject(v) do |memo, i|
+                tmp = current_path.reverse.inject(v) do |memo, i|
```

Comment:
Lazy enumeration here makes 0 sense so to simplify I've got rid and not noticed any difference behaviourally